### PR TITLE
feat(HOME-857): wiring web appearance for consumer v2

### PR DESCRIPTION
**Pull request by antoinemeyer**
@gregdurham that's the schema with the new page structure
required for https://github.com/trizic/platform/pull/971

### CODE REVIEW

File: api/v3/schema/consumer.res.json [hunk 1]
```diff
@@ -1,301 +1,354 @@
...
+					"type": ["string", "null"]
+				},
+				"logo": {
+					"type": ["string", "null"]
+				},
+				"banner": {
+					"type": ["string", "null"]
+				}
+			},
+			"required": ["background", "logoAlt", "logo", "banner"]
```
Comment (gregdurham) [inline api/v3/schema/consumer.res.json]: This is not required, these are optional.

File: api/v3/schema/consumer.res.json [hunk 2]
```diff
@@ -1,301 +1,354 @@
...
+					"type": "object",
+					"properties": {
+						"everydayBg": {
+							"type": ["string", "null"]
+						},
+						"desktop": {
+							"type": ["string", "null"]
+						}
+					},
+					"required": ["everydayBg", "desktop"]
```
Comment (gregdurham) [inline api/v3/schema/consumer.res.json]: Not required

File: api/v3/schema/consumer.res.json [hunk 3]
```diff
@@ -1,301 +1,354 @@
...
+				},
+				"portfolio": {
+					"type": "object",
+					"properties": {
+						"paragraphs": {
```
Comment (gregdurham) [inline api/v3/schema/consumer.res.json]: missing title field.

File: api/v3/schema/consumer.res.json [hunk 4]
```diff
@@ -1,301 +1,354 @@
...
+						"portfolioName": {
+							"type": ["string", "null"]
+						}
+					},
+					"required": ["holdings", "portfolioName"]
+				},
+				"colors": {
+					"type": "object",
+					"properties": {
+						"taglines": {
```
Comment (gregdurham) [inline api/v3/schema/consumer.res.json]: I would keep this generic, tagLines may not be the only option under colors. Should we choose to support more options in the firmadmin, we likely do not want to need to update this repo on every change.

Comment (antoinemeyer) [inline api/v3/schema/consumer.res.json]: I left this one there but made it non-required so we can add similar fields, or not.

**Pull request merged**